src/api/posts/test/posts.test.js

```diff
@@ -69,6 +86,25 @@ describe('/posts', () => {
     expect(res.body.length).toBeGreaterThan(0);
   });
 
+  test('request posts with a valid expand(do not expand) query param', async () => {
+    const res = await request(app).get('/?page=1&expand=0');
+    expect(res.status).toEqual(200);
+    // This will depend on the env value, so as long as we get back something.
+    expect(res.body.length).toBeGreaterThan(0);
+  });
+  test('request posts with a valid expand query param', async () => {
+    const res = await request(app).get('/?page=1&expand=1');
+    expect(res.status).toEqual(200);
+    // This will depend on the env value, so as long as we get back something.
+    expect(res.body.length).toBeGreaterThan(0);
```

Comment:
This isn't enough.  In the case that you include `expand=1`, we'll expect the shape of the objects returned in the array to be different.  Please add some logic to the tests to deal with the case of `expand=1` and without, looking at the properties in the objects.

Reply:
Got it, I've pushed a revised test that now checks the extra properties in the return data when `expand=1`.

Reply:
This is getting better, but still doesn't test what we need.

Use Jest's [`arrayContaining`](https://jestjs.io/docs/expect#expectarraycontainingarray) and [`objectContaining`](https://jestjs.io/docs/expect#expectobjectcontainingobject) to look for one item that has the shape/values you expect in the results returned.